backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventFlux.java

```diff
@@ -42,7 +48,11 @@ public DiagramEventFlux(Diagram currentDiagram) {
 
     public void diagramRefreshed(IInput input, Diagram newDiagram) {
         this.currentDiagram = newDiagram;
-        this.sink.tryEmitNext(new DiagramRefreshedEventPayload(input.getId(), this.currentDiagram));
+        EmitResult emitResult = this.sink.tryEmitNext(new DiagramRefreshedEventPayload(input.getId(), this.currentDiagram));
+        if (emitResult.isFailure()) {
+            String pattern = "An error has occurred while emitting a DiagramRefreshedEventPayload: {0}"; //$NON-NLS-1$
+            this.logger.warn(MessageFormat.format(pattern, emitResult));
```

Comment:
SLF4J supports message formatting natively, and using that support prevents building the string if the logging level invoked is disabled. This kind of pattern should be written like this instead:

```java
this.logger.warn("An error has occurred while emitting a DiagramRefreshedEventPayload: {}", emitResult);
```

Granted, in this particular case (logging an error) it's not much of an improvement, but there are many other places where we should probably use that pattern instead.